### DISCUSSION TITLE
PMM Docker - Incorrect Time

### DISCUSSION THREAD

**Discussion by jarodaustin** [meisnate12/Plex-Meta-Manager#1743] [Q&A] [answered]
I installed PMM using the docker desktop method a few weeks ago, and I was noticing "consistency" issues with the collection scheduling. When I checked the logs, I noticed they were 6 hours ahead of my local computer time. Everywhere I look says I need to update the "docker-compose.yml" file with the correct timezone, but I can't find that file anywhere on my computer. Can anyone help me get the date/time set properly on my PMM docker desktop image?

**Comment by chazlarson** [comment]
How did you create the container?  If you didn't use docker-compose, you won't have a docker-compose.yml file, and if you did you would have created that file, so would presumably know where you did so.

I'm asking because I'm not sure what "the docker desktop method" means.

**Comment by jarodaustin** [comment]
I created it using the "Docker Walkthrough" on the metamanager website: https://metamanager.wiki/en/latest/home/guides/docker.html

**Reply by chazlarson** [reply, marked as answer]
OK; then you should be able to add the timezone to the docker run command:

```
docker run -e TZ=America/New_York ...rest of command...
```

**Reply by jarodaustin** [reply]
That did the trick! My schedule configs are working properly now and the logs are showing the correct date/time.
Thank you!
